components/match2/wikis/hearthstone/match_group_input_custom.lua

```diff
@@ -0,0 +1,435 @@
+---
```

Comment:
not reviewing this file
match input processing will get a big refactor, see #4481 and #4511